builder/gen-dockerfile/contents/src/data/Dockerfile

```diff
@@ -0,0 +1,3 @@
+# Dockerfile extending the generic Node image with application files for a
+# single application.
+FROM gcr.io/google_appengine/nodejs
```

Comment:
Thanks for catching this.  I've updated the code to use the `CANDIDATE_NAME` variable so that the base (runtime) image and the image that is part of the builder pipeline all use the same tag which is the value of `CANDIDATE_NAME`.